tests/wrapper/test_simulation_object_updating_component.py

```diff
@@ -39,15 +39,11 @@ def component(self, traffic_update) -> SimulationObjectUpdatingComponent:
 
         return component
 
-    @pytest.fixture
-    def configured_component(self, traffic_update) -> SimulationObjectUpdatingComponent:
-        # pylint: disable=unused-argument
-        component = SimulationObjectUpdatingComponent(
-            sumo_configuration=os.path.join(
-                "data", "sumo", "example", "sumo-config", "example.scenario.sumocfg"
-            )
-        )
-        return component
+    def test_tracks(self, souc: SimulationObjectUpdatingComponent):
```

Comment:
why do two test track functions exists?

Reply:
I tried to get rid of a few duplicate fixtures, but I left the component fixture. While trying to remove it, I copied the test. It was not intended for it to stay twice.